code/HabitTracker/app/src/test/java/com/example/habittracker/DatabaseManagerTest.java

```diff
@@ -0,0 +1,74 @@
+//package com.example.habittracker;
+//
```

Comment:
Mmm ok so I see a lot of commented out test code here. What happened? Is it failing? Do we need to make an issue for someone to fix this?

Reply:
We are having a connection error to FirestoreApp and that's why the entire script is commented out. We might want to delete this script if the error can't be resolved in time.